services/otel-gateway/.gitignore

```diff
@@ -0,0 +1,4 @@
+exported
```

Comment:
Would this be better in the top-level .gitignore?

Reply:
as in adding `services/otel-gateway/exported` to the top level `.gitignore`?

Why is that preferable?

Reply:
Ah, I was commenting on the whole file, but github doesn't really let me do that. It does make sense to keep `exported` in this file, that's true. `.venv`, `__cache__`, and `.pytest*` might be useful to have at the top level though?

Reply:
Ah, yes will do